src/DevAdventCalendarCompetition/DevAdventCalendarCompetition/Views/Home/Partners.cshtml

```diff
@@ -186,73 +67,20 @@
                     </div>
                 </div>
             </div>
-            <div class="row prizes-wrapper">
-                <div class="col-sm-2 mb-30">
-                    <div class="border prize-wrapper flex-column flex-centered flex-fullheight">
-                        <a href="https://www.facebook.com/fsgeekk" target="_blank" rel="noopener noreferrer">
-                            <img src="~/theme-2019/images/patrons/FSGeek.jpg" alt="fsgeek" title="FSGeek" class="img-responsive center-block" />
-                        </a>
-                    </div>
-                </div>
-                <div class="col-sm-2 mb-30">
-                    <div class="border prize-wrapper flex-column flex-centered flex-fullheight">
-                        <a href="https://www.facebook.com/azuronet" target="_blank" rel="noopener noreferrer">
-                            <img src="~/theme-2019/images/patrons/azuronet.jpg" alt="azuronet" title="Azuronet" class="img-responsive center-block" />
-                        </a>
-                    </div>
-                </div>
-                <div class="col-sm-2 mb-30">
-                    <div class="border prize-wrapper flex-column flex-centered flex-fullheight">
-                        <a href="https://www.facebook.com/azuronet" target="_blank" rel="noopener noreferrer">
-                            <img src="~/theme-2019/images/patrons/netmeetup.jpg" alt="neetmeetup" title=".NET Meetup Poland" class="img-responsive center-block" />
-                        </a>
-                    </div>
-                </div>
-                <div class="col-sm-2 mb-30">
-                    <div class="border prize-wrapper flex-column flex-centered flex-fullheight">
-                        <a href="https://dotnetomaniak.pl/" target="_blank" rel="noopener noreferrer">
```

Comment:
Are we sure to remove dotnetomaniak.pl?

Reply:
I haven't received an info to not remove them. So I did. Can get them back.